generator/md.ts

```diff
@@ -69,8 +69,7 @@ function extractHeadersPlugin(md: markdownit): void {
     if (titleToken.type === "inline") {
       const title =
         titleToken.children
-          ?.filter((t) => t.type === "text")
```

Comment:
My headings had `<code>` tags. This seems to fix it. The text of any tags will be concatenated together. Don't restrict to "text" only, I think that's just text not in tags.

Reply:
99% of that function is GPT4 + copilot xD